src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -14,7 +14,7 @@ const ErrorPage = () => {
         LOOKS LIKE YOU'VE TAKEN A WRONG TURN.
       </h1>
       <ErrorSVG />  
-      <Link to={isAuthenticated ? "/" : "/auth"}>
+      <Link to={isAuthenticated ? "/" : "/"}>
```

Comment:
can't this just be put to / now as that check is saying if true send to / or send to / if not ?

Reply:
Changed this now